anomaly_detector/storage/local_storage.py

```diff
@@ -81,10 +81,13 @@ class LSConfiguration(Configuration):
     """Configuration for local storage."""
 
     # Path to a file containing input data. If the value is '-' the input is expected on STDIN
-    LS_INPUT_PATH = Configuration.LOCAL_DATA_FILE #"validation_data/verification_data.json"
+    LS_INPUT_PATH = ""
```

Comment:
I guess these lines can be removed completely then...

Reply:
This whole class is gone. No need for it. LSConfiguration